src/test/java/seedu/savenus/logic/commands/CommandTestUtil.java

```diff
@@ -28,35 +28,40 @@
  */
 public class CommandTestUtil {
 
-    public static final String VALID_NAME_AMY = "Amy Bee";
-    public static final String VALID_NAME_BOB = "Bob Choo";
-    public static final String VALID_PRICE_AMY = "11111111";
-    public static final String VALID_PRICE_BOB = "22222222";
-    public static final String VALID_DESCRIPTION_AMY = "Amy's Description.";
-    public static final String VALID_DESCRIPTION_BOB = "Bob's Description.";
-    public static final String VALID_CATEGORY_AMY = "CatA";
-    public static final String VALID_CATEGORY_BOB = "CatB";
-    public static final String VALID_TAG_HUSBAND = "husband";
-    public static final String VALID_TAG_FRIEND = "friend";
-    public static final String VALID_OPENING_HOURS_AMY = "0800 2100";
-    public static final String VALID_OPENING_HOURS_BOB = "0900 1000";
-    public static final String VALID_RESTRICTIONS_AMY = "Not halal";
-    public static final String VALID_RESTRICTIONS_BOB = "Contains dairy";
-
-    public static final String NAME_DESC_AMY = " " + PREFIX_NAME + VALID_NAME_AMY;
-    public static final String NAME_DESC_BOB = " " + PREFIX_NAME + VALID_NAME_BOB;
-    public static final String PRICE_DESC_AMY = " " + PREFIX_PRICE + VALID_PRICE_AMY;
-    public static final String PRICE_DESC_BOB = " " + PREFIX_PRICE + VALID_PRICE_BOB;
-    public static final String DESCRIPTION_DESC_AMY = " " + PREFIX_DESCRIPTION + VALID_DESCRIPTION_AMY;
-    public static final String DESCRIPTION_DESC_BOB = " " + PREFIX_DESCRIPTION + VALID_DESCRIPTION_BOB;
-    public static final String CATEGORY_DESC_AMY = " " + PREFIX_CATEGORY + VALID_CATEGORY_AMY;
-    public static final String CATEGORY_DESC_BOB = " " + PREFIX_CATEGORY + VALID_CATEGORY_BOB;
-    public static final String TAG_DESC_FRIEND = " " + PREFIX_TAG + VALID_TAG_FRIEND;
-    public static final String TAG_DESC_HUSBAND = " " + PREFIX_TAG + VALID_TAG_HUSBAND;
-    public static final String OPENING_HOURS_DESC_AMY = " " + PREFIX_OPENING_HOURS + VALID_OPENING_HOURS_AMY;
-    public static final String OPENING_HOURS_DESC_BOB = " " + PREFIX_OPENING_HOURS + VALID_OPENING_HOURS_BOB;
-    public static final String RESTRICTIONS_DESC_AMY = " " + PREFIX_RESTRICTIONS + VALID_RESTRICTIONS_AMY;
-    public static final String RESTRICTIONS_DESC_BOB = " " + PREFIX_RESTRICTIONS + VALID_RESTRICTIONS_BOB;
+    public static final String VALID_NAME_CHICKEN_RICE = "Amy Bee";
+    public static final String VALID_NAME_NASI_LEMAK = "Bob Choo";
+    public static final String VALID_PRICE_CHICKEN_RICE = "11111111";
+    public static final String VALID_PRICE_NASI_LEMAK = "22222222";
```

Comment:
Do you think we should change this to something more food appropriate? Just a suggestion it's not very important

Reply:
I overlooked this oh my goodness. I go change don't merge yet then !!